public/docs/release-notes.md

```diff
@@ -1695,3 +1695,7 @@ otherwise you will encounter login-issues.
 - Realtime collaborate
 - Cross-platformed
 - Recently used history
+
+### Enhancements
+
+- Added yaml metadata doc into hedgedoc frontend from public/docs
```

Comment:
1. The release notes for the newest version go to the top of the file. Not the bottom.
2. This release note isn't necessary because your change doesn't alter the app itself.

Reply:
Reverted changes.